test-config/sensirion_test_setup.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SENSIRION_TEST_SETUP_H
+#define SENSIRION_TEST_SETUP_H
+
+#include <stdio.h>
```

Comment:
Why do we need to include `stdio.h` here?

Reply:
No Idea will look into that when I remove the other function, okay? Added to story.